src/main/java/co/omise/models/TransactionDirection.java

```diff
@@ -2,7 +2,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public enum TransactionType {
+public enum TransactionDirection {
     @JsonProperty("credit")Credit,
     @JsonProperty("debit")Debit,
```

Comment:
Nitpicky detail here, but should it have a space between the declaration?

Reply:
Between `)` and the declaration you mean? Yeah correct

Reply:
Yup